e2e/tests/ui-driven/playwright.config.ts

```diff
@@ -46,7 +46,7 @@ const config: PlaywrightTestConfig = {
   webServer: {
     command: `pnpm ui`,
     port: 3000,
-    reuseExistingServer: false,
+    reuseExistingServer: !process.env.CI,
```

Comment:
This allows us to run the React app at `localhost:3000` when running Playwright tests and makes for a much better feedback loop than make change → build → re-run tests.